File: inventario_python/models/factura.py
class Factura:
    def __init__(self):
        self.id = 0
        self.movimiento_id = None
        self.cliente_id = None
        self.numero_factura = ""
        self.fecha_emision = None
        self.metodo_pago = ""
        self.estado = ""
        self.subtotal = 0.0
        self.impuestos = 0.0
        self.total = 0.0
        self.cliente_nombre = ""

File: inventario_visual/Models/Factura.vb
Namespace inventario_visual
    Public Class Factura
        Public Property Id As Long
        Public Property MovimientoId As Long?
        Public Property ClienteId As Long?
        Public Property NumeroFactura As String
        Public Property FechaEmision As DateTime
        Public Property MetodoPago As String
        Public Property Estado As String
        Public Property Subtotal As Double
        Public Property Impuestos As Double
        Public Property Total As Double
        Public Property ClienteNombre As String
    End Class
End Namespace
